packages/lesswrong/components/ea-forum/giving-portal/GivingSeasonHeader.tsx

```diff
@@ -205,7 +241,60 @@ const styles = (theme: ThemeType) => ({
   },
   disabledStepLink: {
     opacity: 0.7,
-  }
+  },
+  gsContent: {
+    fontFamily: theme.palette.fonts.sansSerifStack,
+    fontSize: 36,
+    fontWeight: 700,
+    lineHeight: "120%",
+    letterSpacing: "-0.76px",
+    display: "flex",
+    flexDirection: "column",
+    gap: "20px",
+    justifyContent: "center",
+    height: "100%",
+    [theme.breakpoints.down("sm")]: {
+      display: "none",
+    },
+  },
+  gsButtons: {
+    display: "flex",
+    gap: "20px",
+    "& .EAButton-variantContained": {
+      background: theme.palette.text.alwaysWhite,
+      fontWeight: 600,
+      zIndex: 3, // Appear on top of any hearts
+    },
+  },
+  gsHearts: {
+    position: "relative",
+    [theme.breakpoints.down("sm")]: {
+      display: "none",
+    },
+  },
+  gsHeart: {
+    position: "absolute",
+    zIndex: 2,
+    color: theme.palette.givingPortal.homepageHeader.main,
+    marginLeft: -12,
+    marginTop: -12,
+  },
+  gsHeartTooltip: {
+    backgroundColor: theme.palette.panelBackground.tooltipBackground2,
+  },
+  gsCanPlaceHeart: {
+    cursor: "none",
+  },
+  gsHeartCursor: {
+    pointerEvents: "none",
+    color: theme.palette.givingPortal.homepageHeader.secondary,
+  },
+  gsAddingHeart: {
+    cursor: "wait !important",
+  },
+  gsRightHeaderItems: {
+    zIndex: 3, // Appear on top of any hearts
```

Comment:
This doesn't quite work, ex. this heart prevents me from clicking on search. Looks like you can just decrease `gsHeart`'s `zIndex`.

<img width="1275" alt="Screen Shot 2023-12-26 at 3 25 09 PM" src="https://github.com/ForumMagnum/ForumMagnum/assets/9057804/0df0d979-eb61-4d5c-bcd1-8d76fdf20a5d">